example/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@discordjs/voicelink_bot",
+	"version": "0.1.0",
+	"description": "",
+	"scripts": {
+		"build": "rimraf dist && tsc",
+		"start": "node dist/index.js",
+		"lint": "eslint src packages --ext .ts --fix"
+	},
+	"keywords": [],
+	"author": "iCrawl <icrawltogo@outlook.com>",
+	"license": "UNLICENSED",
+	"private": true,
+	"dependencies": {
+		"@discordjs/opus": "^0.5.3",
+		"discord.js": "^13.0.0-dev.f7eeccba4b7015496df811f10cc2da2b0fab0630",
```

Comment:
Just wondering is it safe for 13.0.0-dev to be used by everyone? Just making sure (**THIS IS JUST A SUGGESTION / COMMENT**)